src/Stage/Parse/Parser.elm

```diff
@@ -804,6 +805,21 @@ tuple3 config =
         |> located
 
 
+record : ExprConfig -> Parser_ LocatedExpr
+record config =
+    P.succeed Frontend.Record
+        |= P.sequence
+            { start = P.Token "{" ExpectingRecordLeftBrace
+            , separator = P.Token "," ExpectingRecordSeparator
+            , end = P.Token "}" ExpectingRecordRightBrace
+            , spaces = spacesOnly
```

Comment:
I'm wondering about this vs `P.spaces` (but there is quite possibly some trickery with needing the same indentation etc., which is problematic in more places already)

Reply:
We don't have to solve this in this PR - we can do a sweep of "what to do with multi-line expressions" for all expressions in another PR later.

Reply:
yeah, I've just taken the list parser and "did the same", aka "only support one-line data structure" for the moment.